Formula/q/qt.rb

```diff
@@ -28,15 +26,14 @@ class Qt < Formula
     sha256 cellar: :any, arm64_ventura: "f30408e5e2a56aef97597398e4d552da3286b9fb45ba040e405291584009bea8"
     sha256 cellar: :any, sonoma:        "1cb92af62ec86c41bd6901428356fc8908ab98706a1f4b1e7791f09fd6cad498"
     sha256 cellar: :any, ventura:       "3f55e107498966094f5b47f6e729728c7cc97d503f9deed8c59d87ca9f69cfa4"
+    sha256               arm64_linux:   "0f25abc1c0b90ec910b12b3d5c4db85e77e5f0888a2a8446f2d2dc5c973ede78"
```

Comment:
Seeing if possible to build on ARM runner when split. Will drop once done testing
```suggestion
```